test/asset/vocab_test.txt

```diff
@@ -4,4 +4,4 @@ c
 a
 b
 a
-c
\ No newline at end of file
+c
```

Comment:
What exactly is the change for this line? On the Github UI, the diff looks the same to me

Reply:
Hey @Nayef211 this PR does not make any formatting change. The change in `vocab_test.txt` is to add a new line at the end of the file, so that the test is also able to test this condition.